projects/synclub-staked-bnb/index.js

```diff
@@ -0,0 +1,22 @@
+const { get } = require("../helper/http")
+let _res
+
+async function getData() {
+  if (!_res) _res = get("https://www.synclub.io/staas/v1/public/staking/synclub/tvl")
```

Comment:
can you update the adapter to fetch the info onchain?

Reply:
The url is right. It works now.

Reply:
Changed to fetch from chain